src/main/kotlin/com/github/pool_party/pull_party_bot/commands/Utils.kt

```diff
@@ -32,7 +32,8 @@ fun Bot.onAdministratorCommand(command: String, action: (Message, String?) -> Un
 fun Bot.modifyCommandAssertion(chatId: Long, name: String): Boolean =
     name.equals("admins").not().also { if (!it) sendMessage(chatId, ON_ADMINS_PARTY_CHANGE, "Markdown") }
 
-fun parseArgs(args: String?): List<String>? = args?.split(' ')?.map { it.trim().toLowerCase() }?.distinct()
+fun parseArgs(args: String?): List<String>? =
+    args?.split(' ')?.map { it.trim().toLowerCase() }?.filter { it.isNotBlank() }
```

Comment:
Probably it is better to write `args?.let { <other code> }` in order not to write `?.` constantly. But that's ok